editoast/src/views/work_schedules.rs

```diff
@@ -0,0 +1,262 @@
+use actix_web::post;
+use actix_web::web::Data;
+use actix_web::web::Json;
+use chrono::NaiveDateTime;
+use chrono::Utc;
+use derivative::Derivative;
+use editoast_derive::EditoastError;
+use serde::de::Error as SerdeError;
+use serde::Deserialize;
+use serde::Serialize;
+use std::result::Result as StdResult;
+use thiserror::Error;
+use utoipa::ToSchema;
+
+use crate::error::InternalError;
+use crate::error::Result;
+use crate::modelsv2::work_schedules::WorkSchedule;
+use crate::modelsv2::work_schedules::WorkScheduleGroup;
+use crate::modelsv2::work_schedules::WorkScheduleType;
+use crate::modelsv2::Changeset;
+use crate::modelsv2::Create;
+use crate::modelsv2::Model;
+use crate::schema::TrackRange;
+use crate::DbPool;
+
+crate::routes! {
+    "/work_schedules" => {
+        create,
+    }
+}
+
+editoast_common::schemas! {
+    WorkSchedule,
+    WorkScheduleCreateForm,
+    WorkScheduleGroup,
```

Comment:
I think `WorkScheduleGroup` isn't used in openAPI right now

Reply:
Done :white_check_mark: